### ISSUE TITLE
Problem on https://docs.typo3.org/m/typo3/tutorial-sitepackage/13.4/en-us//MinimalExample/Index.html

### ISSUE THREAD

**Issue by BerndSx** [TYPO3-Documentation/TYPO3CMS-Tutorial-SitePackage#288]
We recommend submitting changes and suggestions via pull requests. For small changes you
can use the "Edit on GitHub" button which is found at the top of every documentation
page.

For more information on how you can contribute and help improve TYPO3's
documentation, visit the "How to document guide": https://docs.typo3.org/m/typo3/docs-how-to-document/main/en-us/WritingDocsOfficial/Index.html

### Current behavior

b13/make doesn't support version 13 so far. I experienced that the structure was wrong.
Also the given answers:
 May we add a basic service configuration for you? (yes/no) [yes]:
     > no
    
     May we create a ext_emconf.php for you? (yes/no) [no]:
     >
aren't helpful. The extension didn't appear in my installation.

### Expected behavior/output

A clear and concise description of what you expect to happen.

### Links

One or more links to the documentation page (if available), preferably to the
relevant section. If applicable, also link to other resources which might be
helpful to better understand your proposal.

### TYPO3 versions
13

### Possible Solution

Only if you have concrete suggestions to solve the issue.

### Additional context

Add any other context or screenshots about your proposal here.

**Comment by linawolf** [comment]
According to https://packagist.org/packages/b13/make#0.1.8 b13/make is compatible with TYPO3 13.

did you require the extension via composer? were there any error messages?

**Comment by BerndSx** [comment]
I had problems because after installing the site package with composer it was visible in the extension manager but didn't show up in the site configuration. So I was wondering why. Also most of the folders didn't exist.
My next problem was the sites settings, as you mention a folder site/minimal and then describe the use of site/sitepackage.
I first used sitepackage and then changed to minimal. Somehow I was able to proceed then but I don't understand what the problem was.
Using sitepackagebuilder creates a very different folder structure. So that causes even more trouble to follow the guide.
In general I appreciate this way of documentation very much. May be a non expert might verify it.
Regards Bernd 

Am 5. Dezember 2024 15:42:41 MEZ schrieb Lina Wolf ***@***.***>:
